tests/integration/components/stripe-connect-button-test.js

```diff
@@ -13,6 +13,8 @@ test('it renders all required elements', function(assert) {
   this.set('url', url);
   this.render(hbs`{{stripe-connect-button url=url}}`);
 
-  assert.equal(this.$('.stripe-connect').attr('href'), url);
-  assert.equal(this.$('.stripe-connect').text(), 'Connect with Stripe');
+  this.$('.stripe-connect').as((component) => {
+    assert.equal(component.attr('href'), url);
+    assert.equal(component.text(), 'Connect with Stripe');
+  });
```

Comment:
This one is failing for me. You're gonna have to introduce a page object for it, or, which is perfectly fine as an option, don't switch to using `as`here and open an issue to introduce the page object.

Resolving this followed by a rebase and squash into a single commit is enough to get this merged, I think.